src/plot_api/plot_template.js

```diff
@@ -12,7 +12,56 @@
 var Lib = require('../lib');
 var plotAttributes = require('../plots/attributes');
 
-var TEMPLATEITEMNAME = exports.TEMPLATEITEMNAME = 'templateitemname';
+var TEMPLATEITEMNAME = 'templateitemname';
```

Comment:
I'm usually not a fan of three-word attribute names, but I can't really think of a better alternative, so :+1: